### PR TITLE
[search] Fix NormalizeRank.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

File: search/ranker.cpp [hunk 1]
```diff
@@ -448,22 +448,22 @@ class RankerResultMaker
   {
     switch (type)
     {
-    case Model::TYPE_VILLAGE: return rank /= 1.5;
+    case Model::TYPE_VILLAGE: return rank / 1.5;
     case Model::TYPE_CITY:
     {
       if (m_ranker.m_params.m_viewport.IsPointInside(center))
-        return rank * 2;
+        return base::Clamp(rank * 2, 0, 0xFF);
```
Comment (mpimenov) [inline search/ranker.cpp]: cast rank to something wider, please

Comment (tatiana-yan) [reply search/ranker.cpp]: done

File: search/ranker.cpp [hunk 2]
```diff
@@ -448,22 +448,22 @@ class RankerResultMaker
   {
     switch (type)
     {
-    case Model::TYPE_VILLAGE: return rank /= 1.5;
+    case Model::TYPE_VILLAGE: return rank / 1.5;
     case Model::TYPE_CITY:
     {
       if (m_ranker.m_params.m_viewport.IsPointInside(center))
-        return rank * 2;
+        return base::Clamp(static_cast<int>(rank * 2), 0, 0xFF);
```
Comment (mpimenov) [inline search/ranker.cpp]: static_cast<int>(rank) * 2

Comment (tatiana-yan) [reply search/ranker.cpp]: ok, done

**Pull request merged**